mne/event.py

```diff
@@ -827,3 +828,289 @@ def concatenate_events(events, first_samps, last_samps):
         events_out = np.concatenate((events_out, e2), axis=0)
 
     return events_out
+
+
+class Elekta_event(object):
```

Comment:
found this from an old tweet by @agramfort : https://www.youtube.com/watch?v=o9pEzgHorH0 :) we shouldn't overfit for future design plans ...